src/Http/Controllers/Controller.php

```diff
@@ -157,7 +157,7 @@ public function insertUpdateData($request, $slug, $rows, $data)
     /**
      * Validates bread POST request.
      *
-     * @param \Illuminate\Http\Request $request The Request
+     * @param array                    $request The Request
```

Comment:
Please rename `$request` to `$data` to avoid confusion.  Otherwise I'm good with this